### ISSUE TITLE
Turn on screen and activate certain tab when state of an item is changed...

### ISSUE THREAD

**Issue by TommySharpNZ** [igorgladkov/rotini#75]
I'm thinking of having a tab which shows a number of security cameras around my home. What would be super cool is that when motion is detected by a camera then my tablet on the wall would switch on it's screen and go to the tab that shows the cameras.

I can setup a switch in OpenHAB that will get updated when movement is detected.

So the main requirements would be....
- Being able to switch off the screen after x minutes (android will probably do this anyways)
- Being able to switch on the screen based on an openHAB item being set to "on" or something like that.
- When the screen is switched on, an option to automatically switch to a certain siteman/tab

**Comment by anordvall** [comment]
I need this one as well! I guess the Android screen can not be turned on from the app (?) but rather an in-app screensaver could be turned off when a widget (such as a motion-sensor) would get updated.

If the specific widget would update I would also like to see the screensaver delay get reset, so e.g. the screensaver would start x minutes after the last movement.

Also see #38 for related discussion.

**Comment by frode** [comment]
Technically it's possible to turn on the device screen from the app. It's already being done by Tasker + plugin (secure settings)

I can image that this would be practical for house security, a PIR sensor detects movement and your tablet is near by the entrance, you get an active screen and use Rotini to deactivate the alarm.

**Comment by TommySharpNZ** [comment]
Hey @igorgladkov, you had any more thought on turning the screen off after x seconds/minutes and then back on based on some form of external input or even movement detected by the camera?

I'm about to mount my tablet on the wall so this would be a cool feature for that!
The tablet does have "double tap the screen" to turn on the screen but it would be so cool if it could turn on as I walk past it.

**Comment by smar000** [comment]
@TommySharpNZ You can achieve the screen off/on effect using a combination of `Tasker`and `Motion Detector`, both apps from the Google Play store.

**Comment by anordvall** [comment]
Using Tasker and Motion Detector is still a workaround. It adds additional cost and perhaps even requires a rooted device (?). Also, this request includes possibility to switch to a certain tab (should perhaps also be sitemap) and this is afaik not possiblel through Tasker.

As far as I can see, it should be a fairly quick thing for @igorgladkov  to implement, so I have big hopes :) If Rotini can not actually control screen on/off, at least Rotini can launch its own screensaver. Default should be completely black I would say, but options could be to show a clock, etc etc..

**Comment by tiny-tinker** [comment]
+1 For this. I would love to have this as well. I'm worried that on this cheapy tablet I'll burn the screen if I have it on all the time.

**Comment by spacemanspiff2007** [comment]
My last tablet burnt the screen showing the openhab webpage.
So +1 for the possibility to switch the screen off and on through an item.

**Comment by SierraNL** [comment]
Great idea, for security cams, motion detectors and such, but you could also use it for certain notifications.

**Comment by psixilambda** [comment]
Yes great addition. Could be used for doorbells as well to make sure to show the front camera feed when item state changed to "bell ringing".
